packages/db/src/schema/user.ts

```diff
@@ -59,12 +59,15 @@ export const userDataPropertyEnums: Record<(typeof userData.property.enumValues)
 
 export const userDataHotWaterEnums: Record<(typeof userData.hotWater.enumValues)[number], string> = {
     electric: "Elektrisch",
+
     not_electric: "Nicht elektrisch",
 };
 
-export const mail = mysqlTable("mail", {
+export const reports = mysqlTable("reports", {
```

Comment:
Vllt sollten wir für die reports Tabelle auch noch eine History-Tabelle anlegen (wie für user_data). Dann können wir im Nachhinein sehen, ob und warum die Einstellungen von Benutzern geändert wurden.

Reply:
Doch, das gehört in die DB, damit wir das später vernünftig auswerten können. Für andere Tabellen (z. B. user_data und devices) wird das ja auch so gemacht